### ISSUE TITLE
Problem in Notebook For DINOV3

### ISSUE THREAD

**Issue by rotk27** [tue-mps/eomt#44]
Thanks for your excellent job！ I haven't seen such elegant code in a long time.

However, when I try to use notebook to infer COCO Datasets for instance segmentation, I found that the predicted graph had no results. 

The config file I used is  eomt/configs/dinov3/coco/instance/eomt_large_1280.yaml， I also download corresponding weights file in hugginface. 

The first problem I found is you register pixel_mean and pixel_std in vit.py，but I found that this data in pytorch_model.bin is 0, so when emot try to normalize input, x = (x - self.encoder.pixel_mean) / self.encoder.pixel_std, it didn't work because the value of pixelmean and std is 0.  The original result for class_logits distribution is NAN and mask_logits distribution is also NAN.

After I fixed this problem, the model still output all black image, 

=== mask_logits Distribution ===
shape: torch.Size([1, 200, 160, 160])
range 199.3750 ~ 449.7500
mean: 353.0000, standard: 53.8438

=== class_logits distribution ===
shape: torch.Size([1, 200, 81])
range: -4.3203 ~ 4.2461
mean: 1.9053, standard: 0.9717.

Here is the output image I got.  I also tired panoptic coco dataset and panoptic model , also failed.

<img width="4470" height="1081" alt="Image" src="https://github.com/user-attachments/assets/e83de495-571a-4298-8cee-bdfb70e26e77" />

**Comment by tommiekerssies** [comment]
I just pushed a fix, the notebook should now work for both DINOv2 and DINOv3 EoMT models.
